File: implementation/scipamato/common/wicket/src/test/java/ch/difty/scipamato/common/config/MavenPropertiesTest.java
package ch.difty.scipamato.common.config;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class MavenPropertiesTest {

    private final MavenProperties mp = new MavenProperties();

    @Test
    void noDefaultValues() {
        assertThat(mp.getVersion()).isNull();
        assertThat(mp.getTimestamp()).isNull();
    }
}

File: implementation/scipamato/common/wicket/src/test/kotlin/ch/difty/scipamato/common/config/MavenPropertiesTest.kt
package ch.difty.scipamato.common.config

import org.assertj.core.api.Assertions.assertThat

import org.junit.jupiter.api.Test

internal class MavenPropertiesTest {

    private val mp = MavenProperties()

    @Test
    fun noDefaultValues() {
        assertThat(mp.version).isNull()
        assertThat(mp.timestamp).isNull()
    }
}
